Sources/TuistKit/Commands/TuistCommand.swift

```diff
@@ -25,7 +25,6 @@ public struct TuistCommand: ParsableCommand {
                                  CloudCommand.self,
                                  CacheCommand.self,
                                  SigningCommand.self,
-                                 MigrationCommand.self,
```

Comment:
is this intended as part of this change? this would remove the migration command completely which I don't believe is intended.

Reply:
It doesn't compile locally with this line 🤷‍♂️